### ISSUE TITLE
Horizontal scrollbar in Google Drive and Sheets writer

### ISSUE THREAD

**Issue by MiroCillik** [keboola/kbc-ui#1360]
Unnecessary horizontal scrollbar is shown in Google Drive and Google Sheets writer:
https://monosnap.com/file/fVAqBKoZhVR3i9Nd2xHRJ8KIKBb5Ph

**Comment by janmichek** [comment]
muze souviset -> https://github.com/keboola/kbc-ui/pull/1332

**Comment by ujovlado** [comment]
Ano, suvisi to s tym, viem o tom. Rozbija to ale ina komponenta. Ked si zbalis ten bucket, tak scrollbar zmizne.
